storage/src/main/java/org/apache/uniffle/storage/handler/impl/PrefetchableClientReadHandler.java

```diff
@@ -138,4 +143,13 @@ public void close() {
       prefetchExecutors.shutdown();
     }
   }
+
+  @Override
+  public void logConsumedBlockInfo() {
```

Comment:
Will it be overrided by sub-classes?

Reply:
Checked, all sub-class will call the super. logConsumedBlockInfo